### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
#  Complete website with navigation
- Create a Navigation bar of List, Add New, Contact .
- I don't use any library or framework, use only plain JavaScript and CSS .

### CODE REVIEW

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @HenaryKhuraijam 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights ✔️


- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow
- [x] Nice design.
- [x] Good adding the navigation bar 




## Required Changes ♻️
- Check the comments under the review😊⤵️

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -30,8 +44,32 @@ <h1 class="title-bar">Awesome books</h1>
...
+        <ul class="contact__info">
+          <li>HENARY</li>
+          <li>
+            <a href="https://github.com/HenaryKhuraijam">mybio</a>
+          </li>
+          <li>Zambia, Lusaka</li>
+        </ul>
+      </div>
+    </section>
+  </main>
```
Comment (Zuhaib042) [inline index.html:68]: - [ ] Right now I don't see any content on the main section. and when I click on any link, it doesn't show me the relevant section. This issue was also mentioned by the previous reviewer, kindly consider implementing that.
kindly have a close look at [the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website_v1_1.md) and try to implement them thoroughly.
your project 

https://user-images.githubusercontent.com/96683843/228631885-cdd4ea61-9127-4555-a105-668eced41d27.mov

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @HenaryKhuraijam  @KAYEMBAJONATHAN , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
 No linter errors. 💯
 Good usage in correcting GitHub flow
 Nice design.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @HenaryKhuraijam   and @KAYEMBAJONATHAN ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 
- The SPA is working well :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**